src/SimpleCalculator.Web/Services/CalculatorApiService.cs

```diff
@@ -0,0 +1,40 @@
+using AutoMapper;
+using SimpleCalculator.Web.Models;
+using SimpleCalculator.Web.Services.Interfaces;
+using SimpleCalculator.DataAccess.Data;
+namespace SimpleCalculator.Web.Services;
+
+public class CalculatorApiService : ICalculatorApiService
+{
+    private readonly IMapper _mapper;
+    private readonly CalculatorDbContext _context;
+
+    public CalculatorApiService(CalculatorDbContext context, IMapper mapper)
+    {
+        _context = context;
+        _mapper = mapper;
+    }
+
+
+    public PagingResponseModel GetCalculatorResponse(int page, int pageSize)
+    {
+        var totalCount = _context.CalculationResultEntities.Count();
+        var pageCount = totalCount / pageSize;
```

Comment:
Could you please add a validation that total count is not 0, it would help to avoid extra db call if there is no results in db.